### ISSUE TITLE
User register mail contains two To: header fields

### ISSUE THREAD

**Issue by christianschneider89** [hashtopolis/server#751]
server version from commit a90749126c0af7d6ce3b2b13eb383f6f5a3c5265

When I create a new user, the mail which is sent to the mail server contains two "To:" header fields:

```
From www-data@redacted  Fri Apr  8 11:32:05 2022
Return-Path: <www-data@redacted>
Received: from redacted (localhost [127.0.0.1])
        by redacted (8.15.2/8.15.2/Debian-14~deb10u1) with ESMTP id 2389W5nG016185
        for <redacted@redacted>; Fri, 8 Apr 2022 11:32:05 +0200
Received: (from www-data@localhost)
        by redacted (8.15.2/8.15.2/Submit) id 2389W5pQ016184;
        Fri, 8 Apr 2022 11:32:05 +0200
Date: Fri, 8 Apr 2022 11:32:05 +0200
Message-Id: <202204080932.2389W5pQ016184@redacted>
To: redacted@redacted
Subject: Account at Hashtopolis
MIME-Version: 1.0
From: Hashtopolis <hashtopolis@redacted>
To: redacted@redacted
Content-Type: multipart/alternative;boundary=np625001155b4d5



--np625001155b4d5
Content-type: text/plain;charset=utf-8

Hi redacted,

Your account on Hashtopolis was created. You can now log in with the following credentials:
Username: redacted
Password: redacted
Go to Hashtopolis: http://redacted/hashtopolis

--np625001155b4d5
Content-type: text/html;charset=utf-8

Hi redacted,<br><br>
<p>
        Your account on Hashtopolis was created. You can now log in with the following credentials:<br>
        Username: <code>redacted</code><br>
        Password: <code>redacted</code><br><br>
        Go to Hashtopolis: <a href='http://redacted/hashtopolis'>http://redacted/hashtopolis</a>
</p>

--np625001155b4d5--
```

**Comment by s3inlc** [comment]
Thanks for the report. Indeed, the To: value is included twice because it is provided also in the additional headers passed to the mail() function here: https://github.com/hashtopolis/server/blob/master/src/inc/Util.class.php#L1259

We will fix this.

**Comment by zyronix** [comment]
Should be fixed with #770
